src/common/MessageDispatcher.sol

```diff
@@ -540,23 +540,21 @@ contract MessageDispatcher is Auth, IMessageDispatcher {
     }
 
     /// @inheritdoc IVaultMessageSender
-    function sendUpdateShares(PoolId poolId, ShareClassId scId, D18 pricePoolPerShare, uint128 shares, bool isIssuance)
-        external
-        auth
-    {
+    function sendUpdateShares(PoolId poolId, ShareClassId scId, uint128 shares, bool isIssuance) external auth {
+        // TODO: Remove price in Hub
+        D18 price = d18(1, 1);
```

Comment:
@reviewer 

will be removed in https://github.com/centrifuge/protocol-v3/pull/242